include/uv-win.h

```diff
@@ -78,6 +78,7 @@ typedef intptr_t ssize_t;
 # define SIGABRT_COMPAT       6
 #endif
 
+UV_EXTERN int uv_translate_win32_error(int sys_errno);
```

Comment:
Since this a windows only function, I assume the correct way would be to put it in uv.h and wrap it in `#if defined(_WIN32) ... #endif`?

As for the naming of the functions, it was suggested here: https://github.com/libuv/libuv/issues/79#issuecomment-68108984.